comp/core/workloadmeta/collectors/util/ecs_util.go

```diff
@@ -265,3 +266,16 @@ func parseClusterName(cluster string) string {
 	}
 	return parts[1]
 }
+
+func ParserECSAgentVersion(s string) string {
```

Comment:
Instance metadata returns the version in the format `Amazon ECS Agent - v1.30.0 (02ff320c)`. That's why we need to parse the agent version  http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-introspection.html